cg/cli/post_process/post_process.py

```diff
@@ -35,4 +39,30 @@ def post_process_sequencing_run(context: CGConfig, run_name: str, dry_run: bool)
     post_processing_service.post_process(run_name=run_name, dry_run=dry_run)
 
 
-post_process_group.add_command(post_process_sequencing_run)
+@post_process_group.command(name="all")
+@DRY_RUN
+@click.option(
+    "--instrument",
+    type=click.Choice(["pacbio", "all"]),
+    help="The instrument for which the runs will be post-processed. Choose 'all' to post-process runs from all instruments.",
```

Comment:
Does it make more sense to use a default rather than requiring "all" as input?

Reply:
I have a slight preference for making the field mandatory so that there is no ambiguity, why should there be a default? This command will seldomly be ran manually, it is mostly for the automation

Reply:
My only reason for using a default value is that we will in the future (at least I see it likely that we will) have a constant/enum for all the device types. Without the "all" we can reuse such a constant while we would need to accommodate that "all" is not a device as it is written currently.

Reply:
But go with what you feel is best.

Reply:
I like your point, chenged